lib/utils.ex

```diff
@@ -0,0 +1,26 @@
+defmodule HashRing.Utils do
+  @compile :native
```

Comment:
Does this have any effect?

Reply:
yeah it makes it slightly faster

Reply:
You measured? I've found it to vary pretty wildly, sometimes making things slower.

Reply:
yeah i did. benched w/ and without.